ccmlib/scylla_node.py

```diff
@@ -552,9 +552,10 @@ def import_bin_files(self):
         files = ['cassandra.in.sh', 'nodetool']
         os.makedirs(os.path.join(self.get_path(), 'resources', 'cassandra',
                                  'bin'))
+        tool_dir = os.environ.get('TOOLS_JAVA_DIR', os.path.join(self.get_install_dir(), 'resources', 'cassandra'))
```

Comment:
nit: let's call this variable `tools_java_dir` rather than invent yet another name. It'll be clearer to the reader.

Reply:
Also, can you please extract this change into a separate patch?
It should be valid regardless of the relocatable package

Reply:
Will do